consensus/src/chained_bft/chained_bft_smr.rs

```diff
@@ -138,10 +142,15 @@ impl<T: Payload> ChainedBftSMR<T> {
     /// Create a proposer election handler based on proposers
     fn create_proposer_election(&self) -> Arc<dyn ProposerElection<T> + Send + Sync> {
         assert!(!self.proposers.is_empty());
-        Arc::new(RotatingProposer::new(
-            self.proposers.clone(),
-            self.config.contiguous_rounds,
-        ))
+        match self.config.proposer_type {
+            ConsensusProposerType::MultipleOrderedProposers => {
+                Arc::new(MultiProposer::new(self.proposers.clone(), 2))
```

Comment:
Not sure about contiguous rounds: we used to have it as a crutch to make sure that liveness is guaranteed in certain cases (e.g., 1 out of 4 validators are down). The problem with contiguous rounds is that it increases the p99 latency when one of the validators is down because we'll have several long (timeout) rounds in a row during its period of primary leadership instead of these long rounds being scattered in time.